### PR TITLE
[Website] Version 10.0.1 release blog post

### PR DESCRIPTION
**Pull request by raulcd**

### CODE REVIEW

File: _posts/2022-11-22-10.0.1-release.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+from [**xx distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
```
Comment (raulcd) [inline _posts/2022-11-22-10.0.1-release.md:39]: @lidavidm are there any notes you want to add for the fixes for 10.0.1?

File: _posts/2022-11-22-10.0.1-release.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## C++ notes
```
Comment (raulcd) [inline _posts/2022-11-22-10.0.1-release.md:41]: @pitrou any notes for C++ we should add?

File: _posts/2022-11-22-10.0.1-release.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Go notes
```
Comment (raulcd) [inline _posts/2022-11-22-10.0.1-release.md:43]: @zeroshade any notes for Go we should add to the release blog post for 10.0.1?

File: _posts/2022-11-22-10.0.1-release.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+## Java notes
```
Comment (raulcd) [inline _posts/2022-11-22-10.0.1-release.md:45]: @lidavidm any Java general notes?

File: _posts/2022-11-22-10.0.1-release.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+## Python notes
+
+New features and improvements:
```
Comment (raulcd) [inline _posts/2022-11-22-10.0.1-release.md:49]: @jorisvandenbossche @AlenkaF I've added the Python wheels and the OpenSSL fix, any other notes you want to add?

File: _posts/2022-11-22-10.0.1-release.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+* Added support and wheels for Python 3.11
+  ([ARROW-17487](https://issues.apache.org/jira/browse/ARROW-17487)).
+* Updated OpenSSL bundled on wheels to 3.0.7 due to CVE-2022-3786
+  ([ARROW-18302](https://issues.apache.org/jira/browse/ARROW-18302)).
+
+## R notes
```
Comment (raulcd) [inline _posts/2022-11-22-10.0.1-release.md:56]: @nealrichardson any notes for the 10.0.1 release around R?

File: _posts/2022-11-22-10.0.1-release.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+
+## Ruby and C GLib notes
```
Comment (raulcd) [inline _posts/2022-11-22-10.0.1-release.md:59]: @kou any Ruby notes?

**Review by raulcd — COMMENTED** [review]
@kou I've removed the `community` part. I thought it would be better to add it to the 11.0.0 (major release) release blog post but let me know if you want me to list the new committers/PMC's on this one.

**Comment by raulcd** [comment]
This is the link for the issues on the release:
https://issues.apache.org/jira/issues/?jql=project%20%3D%20ARROW%20AND%20status%20%3D%20Resolved%20AND%20fixVersion%20%3D%2010.0.1

File: _posts/2022-11-22-10.0.1-release.md (hunk 1 above)
Comment (lidavidm) [reply _posts/2022-11-22-10.0.1-release.md:39]:
```suggestion
## Arrow Flight RPC notes
The Arrow Flight SQL JDBC driver now properly handles `executeUpdate` ([ARROW-18294](https://issues.apache.org/jira/browse/ARROW-18294)), and will no longer try to handle URIs that it does not recognize ([ARROW-18296](https://issues.apache.org/jira/browse/ARROW-18296)).
```

File: _posts/2022-11-22-10.0.1-release.md (hunk 4 above)
Comment (lidavidm) [reply _posts/2022-11-22-10.0.1-release.md:45]:
```suggestion
## Java notes
No general changes were made to the Java libraries.
```

File: _posts/2022-11-22-10.0.1-release.md (hunk 7 above)
Comment (kou) [reply _posts/2022-11-22-10.0.1-release.md:61]: 10.0.1 doesn't include any notable changes for Ruby and GLib. Could you remove this part?

**Review by kou — COMMENTED** [review]
> I've removed the `community` part. I thought it would be better to add it to the 11.0.0 (major release) release blog post but let me know if you want me to list the new committers/PMC's on this one.

OK. It makes sense.

File: _posts/2022-11-22-10.0.1-release.md (hunk 3 above)
Comment (raulcd) [reply _posts/2022-11-22-10.0.1-release.md:42]:
```suggestion
## Go notes
* Added option to support dictionary deltas with IPC ([ARROW-18326](https://issues.apache.org/jira/browse/ARROW-18326))
* Fix dictionary replacement during IPC stream ([ARROW-18317](https://issues.apache.org/jira/browse/ARROW-18317))
* Fix StructBuilder premature release fields ([ARROW-18274](https://issues.apache.org/jira/browse/ARROW-18274))
```
@zeroshade does this seem ok?

File: _posts/2022-11-22-10.0.1-release.md (hunk 6 above)
Comment (raulcd) [reply _posts/2022-11-22-10.0.1-release.md:53]:
```suggestion
## R notes
* Fix for failing test after lubridate 1.9 release ([ARROW-18285](https://issues.apache.org/jira/browse/ARROW-18285))
* Add deprecation cycle for pull() change ([ARROW-18132](https://issues.apache.org/jira/browse/ARROW-18132))
* Fix to correctly handle .data pronoun in group_by() ([ARROW-18131](https://issues.apache.org/jira/browse/ARROW-18131))
* Fix for dev purrr ([ARROW-18305](https://issues.apache.org/jira/browse/ARROW-18305))
```
@thisisnic does this seem ok for the 10.0.1 blog post?

File: _posts/2022-11-22-10.0.1-release.md (hunk 2 above)
Comment (raulcd) [reply _posts/2022-11-22-10.0.1-release.md:40]:
```suggestion
## C++ notes
* Add support for ARMv6 ([ARROW-18255](https://issues.apache.org/jira/browse/ARROW-18255))
And some other minor fixes.
```

**Comment by assignUser** [comment]
This should be merged now with the release done? @kou

File: _posts/2022-11-22-10.0.1-release.md (hunk 6 above)
Comment (raulcd) [reply _posts/2022-11-22-10.0.1-release.md:53]: Thanks @thisisnic !

File: _posts/2022-11-22-10.0.1-release.md (hunk 3 above)
Comment (zeroshade) [reply _posts/2022-11-22-10.0.1-release.md:42]: Thanks @raulcd that looks good to me. Sorry for not responding earlier, I was out at the end of last week for Thanksgiving and forgot to update this yesterday.

Comment (raulcd) [reply _posts/2022-11-22-10.0.1-release.md:42]: Thanks @zeroshade !

**Review by kou — APPROVED** [review]
+1

**Pull request merged**